src/main/java/frc/robot/subsystems/Ting.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.hal.simulation.ConstBufferCallback;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.Constants;
+
+public class Ting extends SubsystemBase {
+    TalonFX motor;
+    SingleJointedArmSim arm;
+    Mechanism2d ting1;
+    MechanismLigament2d ligma;
+    public Ting(int motorId){
+        motor = new TalonFX(motorId);
+        motor.setPosition(0);
+        arm = new SingleJointedArmSim(DCMotor.getFalcon500(1) , 1 , .1, .05, Double.NEGATIVE_INFINITY, Double.POSITIVE_INFINITY, false, 0);
+        ting1 = new Mechanism2d(100, 100);
+        ligma = new MechanismLigament2d(getName(), 35, 0);
+        ting1.getRoot("pivot", 50, 50).append(ligma);
+
+
+    }
+
+    public Mechanism2d getMechanism2d(){
+        return ting1;
+    }
+
+    public void periodic(){
+        arm.update(Constants.LOOP_TIME);
+        setMotor(.05);
+        ligma.setAngle(getPos());
```

Comment:
Now that you are using commands to control the motor, you don't need to set the motor periodically anymore.